lib/pwwka/channel_connector.rb

```diff
@@ -67,9 +78,19 @@ def raise_if_delayed_not_allowed
     end
 
     def connection_close
-      channel.close
-      connection.close
-    end
+      begin
+        channel.close
+      rescue => e
+        logf "ERROR Closing RabbitMQ channel", error: e
+        raise e
```

Comment:
I wonder if we should actually not raise back up this & still try to close the connection?

Reply:
yeah, good point.  for the channel errors, we should probably still try to close connection